plugins/carlabase/carla.cpp

```diff
@@ -272,9 +269,6 @@ intptr_t CarlaInstrument::handleDispatcher(const NativeHostDispatcherOpcode opco
     }
```

Comment:
one addition in the code is missing here.
there is a new enum added that needs to be handled, otherwise build against latest carla will fail.
please update your carla first...

Reply:
Won't it generate warnings instead of errors? Does the error occur without `-Werror`?

Reply:
It is a warning yes, but one that can easily be avoided.
we dont want to use older carla with lmms under osx anyway, too many critical bugs in there that were ignored as focused as on linux.

Reply:
The missing enum was for `stable-1.1` only.  This fall through logic on `stable-1.2` doesn't require the explicit enum so it was removed.